user/src/com/google/gwt/user/server/rpc/impl/ServerSerializationStreamWriter.java

```diff
@@ -345,7 +345,7 @@ abstract void write(ServerSerializationStreamWriter stream, Object instance)
    * This exists to work around a Rhino parser bug in the hosted mode client
    * that limits string node lengths to 64KB.
    */
-  private static final int MAX_STRING_NODE_LENGTH = 0xFFFF;
+  private static final int MAX_STRING_NODE_LENGTH =  Integer.getInteger("gwt.rpc.maxStringNodeLength", 0xFFFF);
```

Comment:
(Sorry, reviewdog still isn't supplying annotations automatically...)

> /home/runner/work/gwt/gwt/gwt/user/src/com/google/gwt/user/server/rpc/impl/ServerSerializationStreamWriter.java:348:0: warning: Line is longer than 100 characters (found 111). (com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck)

Reply:
I added the line break